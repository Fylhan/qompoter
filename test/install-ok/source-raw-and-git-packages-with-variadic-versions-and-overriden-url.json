{
	"name": "install-ok/qompoter-versionstar",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"cp/fylhan": "v1.*",
		"fylhan/posibrain": "v0.*"
	},
	"repositories": {
		"fylhan/posibrain": "https://github.com"
	}
}
