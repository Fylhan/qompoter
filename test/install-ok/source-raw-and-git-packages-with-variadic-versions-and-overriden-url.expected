Qompoter
======== install

* cp/fylhan v1.*
  Downloading sources...
  Selected version: v1.1.10
  done

* fylhan/posibrain v0.*
  Downloading sources from Git...
  Selected version: v0.2
  Warning: no 'qompoter.pri' found for this package
  done

done
